src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -138,27 +125,11 @@ public void clickShowStats() {
 
     public void clickIndicateMissingResponses() {
         click(indicateMissingResponsesCheckbox);
-        expandPanels();
     }
 
-    public void expandPanels() {
-        try {
-            // in case the panels are already expanded
-            if (!collapseExpandButton.getText().contains("Expand")) {
-                return;
-            }
-
-            clickCollapseExpand();
-            waitForPageToLoad();
-        } catch (NoSuchElementException e) {
-            // Pages with no feedback response do not have button element
-            // We need to click on each of the panels to expand them
-            List<WebElement> panels = browser.driver.findElements(By.className("panel-heading"));
-            for (WebElement panel : panels) {
-                click(panel);
-            }
-            waitForPanelsToExpand();
-        }
+    public void clickElementById(String elementId) {
```

Comment:
This method looks like it belongs in AppPage more than here since it is a very generic method, can you check if AppPage has it and if it doesn't, add it to AppPage instead of here